src/main/java/de/retest/recheck/suite/flow/ApplyChangesToStatesFlow.java

```diff
@@ -73,7 +73,7 @@ private ApplyChangesToStatesFlow( final Persistence<SutState> persistence ) {
 				return Collections.emptyList();
 			}
 			goldenMasterProvider.saveGoldenMaster( file, newState );
-			return Collections.singletonList( actionChangeSet.getDescription() );
+			return Collections.singletonList( file.getName() );
```

Comment:
Do we want to transform this to a absolute path?

Reply:
I'd say we go with this for now and see how to further improve it later...